ej2-angular/chart/chart-types/line.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: Line Chart in Angular Chart component | Syncfusion
-description: Learn here all about Line Chart in Syncfusion Angular Chart component of Syncfusion Essential JS 2 and more.
+title: Line Chart in Angular Chart Component | Syncfusion
```

Comment:
You can change the title as,
Getting started with Angular Line Chart | Syncfusion

Reply:
changed